### ISSUE TITLE
Account recycler running. Checking status of 0 accounts

### ISSUE THREAD

**Issue by redking974** [PokemonGoMap/PokemonGo-Map#1584]
Hello, I've got this "error" : Account recycler running. Checking status of 0 accounts
I've used this cmd line : "python runserver.py -ac accounts.csv -pd 24"
So is account recycler doesn't work when I load accounts from .csv.
I think that some accounts don't work, but the accounts which works got me resultats (gyms, pokestops, pokemons)

## Expected Behavior
Checking status of my 50 accounts (loaded from .csv file)

## Current Behavior
Account recycler running. Checking status of 0 accounts

## Steps to Reproduce (for bugs)
1. Launch the server
2. Waiting account recycler
3.
4.

## Your Environment
* Version used :
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Windows 10

**Comment by Holzberg** [comment]
I got something similar last weekend.  I have 300 accounts and was running a beehive (pr1367) with -st 3 -w 37 -asi 3600 -ari 3600.  My CSV had 350 accounts and I was using 2captcha.  After about 2 hours scanning would stop and after looking at the logs the recycler was waiting for accounts to cool off.  So i changed -ari to 1000 and still got the same thing.  Finally I changed -ari to 0 and still the same thing.  It seems like the recycler was only using double the workers I had (although I can't 100% confirm this).  Definitely though it was only using a portion of the accounts I had listed.  I checked the csv to make sure there wasn't anything wrong and I know the accounts worked because I was using them with 2 beehive instances -st 2 -w 19 -asi 3600 -ari 3600.  So it seems with the instance with more workers the recylcer had issues. @mdnpascual posted the same thing last night in pr1367, however I never tried a different pattern so I can't confirm it is just with the beehive.
